edogbosunny/team-hemdal-simple-crud
# Chore Setup Travis CI add a travis.yml config

#### What does this PR do?
This PR sets up Travis CI

#### Description of Task to be completed?
- Create .travis.yml file
- Add a coverage, cover, and test line to scripts section in package.json

#### How should this be manually tested?
- Open a terminal window/tab
- Clone this repository.
- Install dependencies with npm install
- Create a new branch, make changes, push and make a PR
- Observe the Continuous Integration checks.

#### What are the relevant pivotal tracker stories?
